roles/docker/docker-storage/defaults/main.yml

```diff
@@ -3,6 +3,9 @@ docker_container_storage_setup_version: v0.6.0
 docker_container_storage_setup_profile_name: kubespray
 docker_container_storage_setup_storage_driver: devicemapper
 docker_container_storage_setup_container_thinpool: docker-pool
+docker_container_storage_setup_vg: docker
+#example docker_container_storage_setup_devs: /dev/vdb
+docker_container_storage_setup_devs:
```

Comment:
This is invalid yaml syntax (A key with no value)

Reply:
ok  ，It must be define a disk path.